e2e/config/config_test.go

```diff
@@ -0,0 +1,130 @@
+package config_test
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/ramendr/ramen/e2e/config"
+	"github.com/ramendr/ramen/e2e/types"
+)
+
+func TestValidatePVCSpecs(t *testing.T) {
+	// storageClass name exceeding 253 chars
+	longSCName := strings.Repeat("a", 254)
+
+	tests := []struct {
+		name    string
+		config  *types.Config
+		wantErr bool
+	}{
+		{
+			name: "empty PVCSpecs list",
+			config: &types.Config{
+				PVCSpecs: []types.PVCSpecConfig{},
+			},
+			wantErr: true,
+		},
+		{
+			name: "valid PVCSpecs",
+			config: &types.Config{
+				PVCSpecs: []types.PVCSpecConfig{
+					{Name: "rbd", StorageClassName: "standard", AccessModes: "ReadWriteOnce"},
+				},
+			},
+			wantErr: false,
+		},
+		{
+			name: "duplicate PVCSpec names",
+			config: &types.Config{
+				PVCSpecs: []types.PVCSpecConfig{
+					{Name: "rbd", StorageClassName: "standard", AccessModes: "ReadWriteOnce"},
+					{Name: "rbd", StorageClassName: "standard", AccessModes: "ReadWriteOnce"},
+				},
+			},
+			wantErr: true,
+		},
+		{
+			name: "duplicate PVCSpec content (same StorageClassName and AccessModes)",
+			config: &types.Config{
+				PVCSpecs: []types.PVCSpecConfig{
+					{Name: "rbd", StorageClassName: "standard", AccessModes: "ReadWriteOnce"},
+					{Name: "cephfs", StorageClassName: "standard", AccessModes: "ReadWriteOnce"},
+				},
+			},
+			wantErr: true,
+		},
+		{
+			name: "invalid storageClassName format - uppercase",
+			config: &types.Config{
+				PVCSpecs: []types.PVCSpecConfig{
+					{Name: "rbd", StorageClassName: "Standard", AccessModes: "ReadWriteOnce"},
+				},
+			},
+			wantErr: true,
+		},
+		{
+			name: "invalid storageClassName format - contains underscore",
```

Comment:
We can remove this test, we don't implement validation,  and the validation library has its own tests.